pull_request_template.md

```diff
@@ -1,52 +1,52 @@
-## Purpose

```

Comment:
Why did you commit this file? I think we do not need to do any changes to this file

Reply:
Fixed with fc5da73f679a393477cb004a5a00da686754b4ed